app/views/curation_concerns/base/_form.html.erb

```diff
@@ -34,7 +34,7 @@
       </div>
 
       <div id="savewidget" class="col-xs-12 col-sm-4 fixedsticky" role="complementary">
-        <%= render 'form_progress', f: f %>
+        <%= render 'form_progress', f: f, agreement_accepted: @form.agreement_accepted %>
```

Comment:
Do we want to make `#agreement_accepted` a model property? At this point I am calculating the value when the WorkForm is instantiated.